### DISCUSSION TITLE
[@types/css-font-loading-module] 0.0.8 fails to build for me

### DISCUSSION THREAD

**Discussion by andybalaam** [DefinitelyTyped/DefinitelyTyped#64302] [Issues with a @types package]
Hi, when I try to build my project [matrix-react-sdk](https://github.com/matrix-org/matrix-react-sdk) with the @types/css-font-loading-module dependency changed to version 0.0.8 (e.g. [this PR](https://github.com/matrix-org/matrix-react-sdk/pull/10132)) I see this build error:

```
$ tsc --noEmit --jsx react && tsc --noEmit --jsx react -p cypress
Error: node_modules/@types/css-font-loading-module/index.d.ts(23,9): error TS2717: Subsequent property declarations must have the same type.  Property 'display' must be of type 'string', but here has type 'any'.
Error: node_modules/@types/css-font-loading-module/index.d.ts(23,19): error TS2304: Cannot find name 'FontDisplay'.
Error: node_modules/@types/css-font-loading-module/index.d.ts(43,18): error TS2304: Cannot find name 'FontDisplay'.
error Command failed with exit code 2.
```

Looking at the code, I can't see where `FontDisplay` comes from, so I suspect this is a real error.

[a comment by an automated account is not shown]

**Comment by JayAitch** [comment]
Hi have a similar issue here, looks to be a redeclaration of FontFaceDescriptors with slightly different types. Typescript 5 and @types/css-font-loading-module are defining the global as a different type :(

**Reply by JayAitch** [reply]
to fix the first error you can in fact add   "skipLibCheck": true to your tsconfig which might be bad practice depending on your school of thought.

**Comment by yordan-kanchelov** [comment]
Fixed the issues by executing once:

npm install --save-dev @types/css-font-loading-module
